.github/ISSUE_TEMPLATE/other-issue.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Other issue
+about: Use this issue template to describe a current issue (that is not a new example analysis)
+title: 'Other issue:'
```

Comment:
If we make the change above, perhaps we should also propagate the change here?
```suggestion
title: 'Other issue re existing analysis:'
```

Or something like that

Reply:
I think the description should be enough.

Reply:
Do you think we need the `Other issue:`? It seems like these will not necessarily have a unifying theme and appending `Other issue:` to the start will make automagically finding them to mention them on GH or connect things via ZenHub more difficult.

Reply:
Ah I see. I'll delete this.